src/components/Article.jsx

```diff
@@ -0,0 +1,42 @@
+import ArticleInfo from "./ArticleInfo"
+import { useParams } from "react-router-dom"
+import axios from "axios"
+import { useEffect, useState} from "react"
+
+
+export default function Article() {
+    const {article_id} = useParams()
+
+
+    let [article, setArticle] = useState()
+    let [loadingArticle, setLoadingArticle] = useState(true)
+    let [comments, setComments] = useState()
+
+    useEffect(() => {
+        setLoadingArticle(true)
+        axios.get(`https://nc-news-backend-shahbazrafi.herokuapp.com/api/articles/${article_id}`)
```

Comment:
This works great, though would be a bit neater if all API calls were kept in their own utils file.
You can also then use axios' baseURL property to neaten up API calls
There's an example of this in the Parametric Endpoint section of the React Router notes if you would like to implement